FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -26,20 +26,28 @@
  */
 
 /*
- * Demo for showing how to use the Device Defender library's API. This version
- * of the Device Defender API provides macros and helper functions for
- * assembling MQTT topics strings, and for determining whether an incoming MQTT
- * message is related to device defender. The Device Defender library does not
- * depend on a MQTT library, therefore the code for MQTT connections is placed
- * in another file (mqtt_demo_helpers.c) to make it easy to read the code using
- * the Device Defender library.
+ * Demo for showing how to use the Device Defender library's APIs. The Device
+ * Defender API provides macros and helper functions for assembling MQTT topics
+ * strings, and for determining whether an incoming MQTT message is related to
+ * device defender. The Device Defender library does not depend on any particular
+ * MQTT library, therefore the code for MQTT operations is placed in another file
+ * (mqtt_demo_helpers.c). This demo uses the coreMQTT library. If needed,
+ * mqtt_demo_helpers.c can be modified to replace coreMQTT with another MQTT
+ * library. This demo requires using the AWS IoT broker as Device Defender is an
+ * AWS service.
  *
  * This demo connects to the AWS IoT broker and subscribes to the device
  * defender topics. It then collects metrics for the open ports and sockets on
  * the device using FreeRTOS+TCP, and generates a device defender report. The
  * report is then published, and the demo waits for a response from the device
  * defender service. Upon recieving the response or timing out, the demo
  * finishes.
+ *
+ * This demo just sets the report ID to 1, which should not be used in
+ * production. Reports for a Thing with a previously used report ID will be
+ * assumed to be duplicates and discarded by the Device Defender service.
+ * The report ID needs to be unique per report sent with a given Thing.
+ * We reccomend using an increasing unique id such as the current timestamp.
```

Comment:
```suggestion
 * We recommend using an increasing unique id such as the current timestamp.
```

Reply:
Please ensure all the spellings in this demo are correct. There are plugins in IDE that you can make use of for spell check